client/src/app/metadata-schema-form/metadata-field-types/base-input/base-input.component.ts

```diff
@@ -34,16 +34,16 @@ export class BaseInputComponent implements OnInit {
     const userFriendly = this.metadata.schema.user_friendly;
     const title = this.metadata.schema.title;
 
-
     const guidelines = this.metadata.schema.guidelines;
     const description = this.metadata.schema.description;
 
     // TODO consolidate form item data
     this.label = userFriendly ? userFriendly : title ? title : this.metadata.key;
-    this.helperText = guidelines ? guidelines : description;
+    const defaultHelperText = description && guidelines ? `${description}<br/><br/>${guidelines}` : description || guidelines || '';
+    this.helperText = this.metadata.guidelines || defaultHelperText;
```

Comment:
The variable `metadata.guidelines` will contain the guidelines set from the config (overrideGuidelines). If it's not set, it will get the value of the helper text from the metadata schema (description and/or guidelines).

Reply:
I refactored it, is it any better?